plugins/php/php_fpm_process

```diff
@@ -140,19 +182,48 @@ total.label Total processes
 total.type GAUGE
 total.draw LINE2
 total.info The number of idle + active processes
+max.label Max processes
+max.type GAUGE
+max.draw LINE
+max.info The maximum number of processes
```

Comment:
This is not very clear to me. Maybe expand it with `... since the start of the FPM process` (if this is what is meant).